### PR TITLE
Prepare for release v5.1.0

### PR DESCRIPTION
**Pull request by andybotting**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-smokeping",
-  "version": "5.0.1-rc0",
+  "version": "5.0.1",
```
Comment (igalic) [inline metadata.json:3]: unless this release was literally a single bug fix, i'd bump the version to 5.1.0

**Review by igalic — CHANGES_REQUESTED** [review]
🙋🏻‍♀️

File: metadata.json (hunk 1 above)
Comment (andybotting) [reply metadata.json]: Updated now to use v5.1.0

**Review by igalic — APPROVED** [review]

**Comment by andybotting** [comment]
:tada:

**Comment by andybotting** [comment]
Thanks!

**Comment by igalic** [comment]
released 5.1.0 https://forge.puppet.com/puppet/smokeping/readme

**Pull request merged**